src/main/java/com/meilisearch/sdk/SearchRequest.java

```diff
@@ -361,6 +459,9 @@ String getQuery() {
         if (this.filter != null) {
             jsonObject.put("filter", this.filter);
         }
+        if (this.filterArray != null) {
+            jsonObject.put("filter", this.filterArray);
+        }
```

Comment:
I would ideally like to throw an exception if both `filter` and `filterArray` were set, but we could also just document it and ignore this or ignore it completely also